app/ui/archiveTile.js

```diff
@@ -180,7 +177,7 @@ module.exports.wip = function(state, emit) {
   <article class="h-full flex flex-col bg-white z-20">
     ${list(
       state.archive.files.map(f => fileInfo(f, remove(f))),
-      'list-reset h-full overflow-y-scroll p-4 bg-blue-lightest max-h-half',
+      'list-reset h-full overflow-y-scroll p-4 bg-blue-lightest md:max-h-half-screen',
```

Comment:
I let the responsive view be full height so that on mobile browsers we don't end up with two scroll areas competing